apps/condo/domains/notification/schema/SyncRemoteClientService.js

```diff
@@ -26,7 +26,7 @@ const SyncRemoteClientService = new GQLCustomSchema('SyncRemoteClientService', {
         },
         {
             access: true,
-            type: 'input SyncRemoteClientInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, appId: String!, pushToken: String, pushTransport: PushTransportType, devicePlatform: DevicePlatformType, pushType: PushType, meta: JSON }',
+            type: 'input SyncRemoteClientInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, appId: String!, pushToken: String, pushTransport: PushTransportType, devicePlatform: DevicePlatformType, pushType: PushType, meta: JSON, pushTokenVoIP: String, pushTransportVoIP: PushTransportType, pushTypeVoIP: PushType }',
```

Comment:
`appId` = mobile app id?

Reply:
not only mobile, in some future we possibly will add for pushes on web.